accord-core/src/main/java/accord/impl/CommandTimeseries.java

```diff
@@ -130,12 +130,12 @@ public Timestamp minTimestamp()
         {
             if (result == null)
             {
-                result = Timestamp.min(loader.txnId(data), loader.executeAt(data));
+                result = Timestamp.nonNullOrMin(loader.txnId(data), loader.executeAt(data));
```

Comment:
also fixed this in [`https://issues.apache.org/jira/browse/CASSANDRA-18804](https://issues.apache.org/jira/browse/CASSANDRA-18804) but hit it in burn tests so fixed in this patch as well.